README.md

```diff
@@ -51,72 +62,130 @@ jobs:
   with:
     projectBaseDir: app/src
     args: >
+      -Dsonar.organization=my-organization # For SonarQube Cloud only
+      -Dsonar.projectKey=my-projectkey
       -Dsonar.python.coverage.reportPaths=coverage.xml
+      -Dsonar.sources=lib/
       -Dsonar.tests=tests/
+      -Dsonar.test.exclusions=tests/**
       -Dsonar.verbose=true
 ```
 
-More information about possible analysis parameters can be found in [the documentation](https://redirect.sonarsource.com/doc/analysis-parameters.html).
+More information about possible analysis parameters can be found:
+* in the [Analysis parameters page](https://docs.sonarsource.com/sonarqube-server/latest/analyzing-source-code/analysis-parameters/) of the SonarQube Server documentation
+* in the [Analysis parameters page](https://docs.sonarsource.com/sonarqube-cloud/advanced-setup/analysis-parameters/) of the SonarQube Cloud documentation
 
 ### Environment variables
 
-- `SONAR_TOKEN` – **Required** this is the token used to authenticate access to SonarQube. You can read more about security tokens [here](https://docs.sonarqube.org/latest/user-guide/user-token/). You can set the `SONAR_TOKEN` environment variable in the "Secrets" settings page of your repository, or you can add them at the level of your GitHub organization (recommended).
-- `SONAR_HOST_URL` – **Required** this tells the scanner where SonarQube is hosted. You can set the `SONAR_HOST_URL` environment variable in the "Variables" settings page of your repository, or you can add them at the level of your GitHub organization (recommended).
-- `SONAR_ROOT_CERT` – Holds an additional root certificate (in PEM format) that is used to validate the SonarQube server certificate. You can set the `SONAR_ROOT_CERT` environment variable in the "Secrets" settings page of your repository, or you can add them at the level of your GitHub organization (recommended).
+- `SONAR_TOKEN` – **Required** this is the token used to authenticate access to SonarQube. You can read more about security tokens in the documentation of SonarQube [Server](https://docs.sonarsource.com/sonarqube-server/latest/user-guide/managing-tokens/) and [Cloud](https://docs.sonarsource.com/sonarqube-cloud/managing-your-account/managing-tokens/). You can set the `SONAR_TOKEN` environment variable in the "Secrets" settings page of your repository, or you can add them at the level of your GitHub organization (recommended).
+- *`GITHUB_TOKEN` – Provided by Github (see [Authenticating with the GITHUB_TOKEN](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/authenticating-with-the-github_token)).*
+- `SONAR_HOST_URL` – **Required** this tells the scanner where SonarQube is hosted.
+  - For SonarQube Server: you can set the `SONAR_HOST_URL` environment variable in the "Variables" settings page of your repository, or you can add them at the level of your GitHub organization (recommended).
+  - For SonarQube Cloud: it should be set to `https://sonarcloud.io`.
```

Comment:
This is the default for new scanners, so I think you can drop this line.

Reply:
Oh, I didn't know, that's great! I have changed to text to reflect that.

This simplifies the migration of users from `sonarcloud-github-action` to `sonarqube-scan-action`, as, apart from the change of action, it should work out of the box without any argument or env var change, so it's basically a drop-in replacement!